first-order-odes/laplace-transform.md

```diff
@@ -1 +1,96 @@
 # Laplace transform
+
+```{example} Laplace transform
+Solve the initial value problem
+
+\begin{equation}
+y' =-ky, \quad y(0) =2
+\end{equation}
+
+---
+
+Rearrange, apply Laplace transform $Y = L[y]$, and solve for $Y$:
+
+\begin{align}
+y'+ ky &= 0 \\
+L[y'+ky] &= L[0] \\
+L[y']+kL[y] &= L[0] \\
+sY - y(0) + k Y &= 0 \\
+(s+k) Y - 2 &= 0 \\
+Y &= \frac{2}{s+k}
+\end{align}
+
+Invert Laplace transform:
+
+\begin{equation}
+y = L^{-1}\left[\frac{2}{s+k}\right] = 2L^{-1}\left[\frac{1}{s+k}\right] = 2 e^{-kt}
+\end{equation}
+```
+
+```{example} Laplace transform with partial fractions
```

Comment:
Please go through this example and make changes to match what I did to the first one!